docs/rules/newline-per-chained-call.md

```diff
@@ -0,0 +1,97 @@
+# Newline Per Chained Method Call (newline-per-chained-call)
+
+This rule enforces new line after each method call when chaining the method calls.
```

Comment:
This introduction needs to be longer. It should show examples and discuss why anyone cares. See http://eslint.org/docs/rules/no-path-concat as an example.